### DISCUSSION TITLE
Brush cursor?

### DISCUSSION THREAD

**Discussion by 6xolotl** [keiwando/toonsquid#183] [Q&A] [answered]
Hi, new user here!! Just got the app today, things are going a lil rough figuring it out but I’m enjoying it so far!!

I wanted to ask if there’s a way to enable a brush cursor. Y’know, the lil grey outline that tells you where your brush is. It’s a small thing, but it helps me a lot. Does ToonSquid have that too?

Thank you!! ^^

**Comment by Cat-Butler** [comment, marked as answer]
Actions > Settings > Tools > Cursor enabled (brush tool) - on/off switch. 
Also for eraser (which I find especially useful) and smudge tools.

**Reply by 6xolotl** [reply]
Thank you!!! ☺️
